### PR TITLE
Added Today button to the datepicker (master)

### PR DESCRIPTION
**Pull request by Pijuli**
Same as #65 but for master branch

### CODE REVIEW

File: src/app/systelab-components/datepicker/README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ All the input parameters are optional.
 | markPreviousAfterDate | boolean | false | If true and currentDate previous than today then the input background color is red |
 | inputExpandHeight | boolean | false | If true the component expands to parent height |
 | fontSize | number | | Font size in pixels |
+| showTodayButton | boolean | true | Show Today button |
```
Comment (alfonsserra) [inline src/app/systelab-components/datepicker/README.md]: Set showTodayButton to false in order to have the same behaviour than before

File: src/app/systelab-components/datepicker/datepicker.component.ts [hunk 2]
```diff
@@ -29,6 +29,7 @@ export class Datepicker implements OnInit, AfterViewInit, DoCheck, OnDestroy {
 	@Input() public inputExpandHeight: boolean;
 	@Input() public markPreviousAfterDate = false;
 	@Input() public inputFontSize: number;
+	@Input() public showTodayButton = true;
```
Comment (alfonsserra) [inline src/app/systelab-components/datepicker/datepicker.component.ts]: Set showTodayButton to false in order to have the same behaviour than before

Comment (Pijuli) [reply src/app/systelab-components/datepicker/datepicker.component.ts]: Done

File: src/app/systelab-components/datepicker/README.md (hunk 1 above)
Comment (Pijuli) [reply src/app/systelab-components/datepicker/README.md]: Done

**Pull request merged**